packages/amplify-category-geo/resources/custom-device-location-tracking-resource-handler.js

```diff
@@ -0,0 +1,30 @@
+const response = require('cfn-response');
```

Comment:
can this be TS?
https://docs.aws.amazon.com/lambda/latest/dg/typescript-handler.html

Reply:
Keeping in line with the other files as JS for now, can follow up in a separate pr to convert them all to TS if that's alright with you